Makefile

```diff
@@ -56,7 +56,7 @@ go_build       ?= $(env) $(go_cmd) build $(build_flags)
 go_test        ?= $(go_cmd) test -json -v -timeout 30m
 proto_path     ?=
 
-ifneq ($(wildcard "/usr/include"),"")
+ifneq ("$(wildcard /usr/include)","")
```

Comment:
That's why `make proto` printed warning for non-existence of this directory.

Reply:
In Linux, it resides in `/usr/include`. In Mac, I don't know. But we don't have to manually set this in Mac.